airbyte-integrations/connectors/source-my-hours/manifest.yaml

```diff
@@ -0,0 +1,1136 @@
+version: 5.14.0
```

Comment:
```suggestion
version: 5.12.0
```

Reply:
Just recommending we change this to 5.12.0 as the builder will complain about the manifest version exceeding the CDK version. This shouldn't actually be a problem (especially since these connectors aren't builder-compatible yet), but might as well play it safe.